packages/cms-cli/bin/cli.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
```

Comment:
When we're fully switched over all the other files in `cms-cli/bin` will be able to be deleted.